holdem/env.py

```diff
@@ -476,6 +471,7 @@ def _get_current_reset_returns(self):
 
   def _get_current_step_returns(self, terminal):
     obs = self._get_current_state()
-    # TODO, make this something else?
-    rew = [player.stack for player in self._seats]
-    return obs, rew, terminal, [] # TODO, return some info?
+    rew = [player.stack - player.hand_starting_stack + player.blind if terminal else 0 for player in self._seats]
```

Comment:
So it looks like this reward indicates how much the player has improved their stack based on what they started the hand with, which sounds fine in theory, but practice can be a little different. I'm not going to split hairs over this, but I'm just wondering - do you have any results indicating that this is preferable?

Reply:
That's how equity is calculated in poker. Expected value of a fold is always 0. And maximizing our expected value is what we should always do.

In real poker a lot of hands are folded preflop and blinds make a huge difference for the winrates. Stealing blinds is the only reason why you shouldn't just wait for AA and go all in with it. So the reward should definitely take this into account.